web_monitoring/html_diff_render.py

```diff
@@ -187,8 +187,16 @@
 MAX_SPACERS = 2500
 
 
+class Strict_url_rule:
+    mode = False
+    REGEX_RULES = {'WBM': re.compile(r"/web/\d{14}"),
+                   'jsessionid': re.compile(r";jsessionid[^;=]*=([^;]+)"),
+                   'UKWA': re.compile(r"https://www\.webarchive\.org\.uk/wayback/en/archive/\d{14}mp_/")}
```

Comment:
Could you please include comments or docstrings describing what each of these are meant to do? We both know the purpose here, but it might not be clear to someone else that:

- `WBM` stands for Wayback Machine
- The use-case here is in comparing Wayback mementos, where links will have been rewritten to point to another memento in Wayback instead of the original URL.
- etc.

^ Same sort of issues for each rule here, not just `WBM`. I know why we need to consider `jsessionid`, but someone else might not.

Reply:
Added